test/tests/materials/torchscript_material/generate_net.py

```diff
@@ -0,0 +1,20 @@
+import torch
+import torch.nn as nn
+
+torch.manual_seed(42)
```

Comment:
In my experience manual_seed isn't reproducible on different architectures. As long as CI doesn't explicitly run this python script it should be fine. But I think in the long term it's better to check in a torch script with saved weights, rather than relying on a python script to generate the network.

Reply:
Oh wow, that's interesting. Thanks for the warning! Yeah right now we are checking in the script file as well.

Reply:
I'm using `manual_seed` in swift tests that run on CPU and CUDA and have not had any issues. In fact, I think this even passes on MPS (I'll double check on monday). Also see https://pytorch.org/docs/stable/notes/randomness.html